doc/development-guide.md

```diff
@@ -74,7 +74,7 @@ Run the following command to dump all keys and values in etcd to stdout. Replace
 different ip and port.
 
 ```sh
-ETCDCTL_API=3 etcdctl get --prefix=true "" --endpoints=[127.0.0.1:2379]
+ETCDCTL_API=3 etcdctl get --prefix=true "" --endpoints="127.0.0.1:2379"
```

Comment:
I'm not sure why this one didn't work for you. All 3 syntax seem to work for me:

```sh
[ppai@gd2-1 glusterd2]$ ETCDCTL_API=3 etcdctl get --prefix=true "gluster-ae83875a-d463-4da2-b770-ad7c5eadf4ab/peers" --endpoints=[127.0.0.1:2379]
gluster-ae83875a-d463-4da2-b770-ad7c5eadf4ab/peers/c9fa8750-4038-4cb0-9666-7069afadae3f
{"ID":"c9fa8750-4038-4cb0-9666-7069afadae3f","Name":"gd2-1","PeerAddresses":["127.0.0.1:24008"],"ClientAddresses":["127.0.0.1:24007"],"MetaData":null}


[ppai@gd2-1 glusterd2]$ ETCDCTL_API=3 etcdctl get --prefix=true "gluster-ae83875a-d463-4da2-b770-ad7c5eadf4ab/peers" --endpoints="127.0.0.1:2379"
gluster-ae83875a-d463-4da2-b770-ad7c5eadf4ab/peers/c9fa8750-4038-4cb0-9666-7069afadae3f
{"ID":"c9fa8750-4038-4cb0-9666-7069afadae3f","Name":"gd2-1","PeerAddresses":["127.0.0.1:24008"],"ClientAddresses":["127.0.0.1:24007"],"MetaData":null}


[ppai@gd2-1 glusterd2]$ ETCDCTL_API=3 etcdctl get --prefix=true "gluster-ae83875a-d463-4da2-b770-ad7c5eadf4ab/peers" --endpoints=127.0.0.1:2379
gluster-ae83875a-d463-4da2-b770-ad7c5eadf4ab/peers/c9fa8750-4038-4cb0-9666-7069afadae3f
{"ID":"c9fa8750-4038-4cb0-9666-7069afadae3f","Name":"gd2-1","PeerAddresses":["127.0.0.1:24008"],"ClientAddresses":["127.0.0.1:24007"],"MetaData":null}
```

Reply:
for me, it's not working


[root@etcd2 ~]# ETCDCTL_API=3 etcdctl get --prefix=true "" --endpoints=[192.168.122.252:2379]
Error: dial tcp: address [192.168.122.252:2379]: missing port in address
[root@etcd2 ~]# 
[root@etcd2 ~]# 
[root@etcd2 ~]# ETCDCTL_API=3 etcdctl get --prefix=true "" --endpoints="192.168.122.252:2379"
gluster-085680b4-7855-4375-ab70-af6322add8a0/alive/45035e69-cbac-4b60-8c65-cf4476940b81


[root@etcd2 ~]#etcdctl --version
etcdctl version: 3.3.4
API version: 2

Reply:
Okay, I'm on an older version:

```
$ etcdctl --version
etcdctl version: 3.2.4
API version: 2
```